index.js

```diff
@@ -13,3 +13,5 @@ const pagination = document.querySelector('[data-js="pagination"]');
 const maxPage = 1;
 const page = 1;
 const searchQuery = "";
+
+cardContainer.append(newCard);
```

Comment:
As already discussed, this line has no function yet as you do not call the `createCharacterCard` to create a new card.